adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequestState.java

```diff
@@ -31,12 +31,15 @@
     private boolean mCancelled = false;
 
     private AuthenticationRequest mRequest = null;
-    
+
+    private final APIEvent mAPIEvent;
+
     public AuthenticationRequestState(int requestCallbackId, AuthenticationRequest request,
-            AuthenticationCallback<AuthenticationResult> delegate) {
+            AuthenticationCallback<AuthenticationResult> delegate, final APIEvent apiEvent) {
```

Comment:
i don't want to repeat this code, adding it here keeps the API surface consistent.

---
In reply to: [76135343](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76135343) [](ancestors = 76135343)